{"name": "test_simple_assume10", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n2 Failed Assumptions:\n\ntest_xdist.py:51: AssumptionFailure\n>>\tpytest.assume(1==0)\nAssertionError: assert False\n\ntest_xdist.py:53: AssumptionFailure\n>>\tpytest.assume(False)\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               2 Failed Assumptions:\nE               \nE               test_xdist.py:51: AssumptionFailure\nE               >>\tpytest.assume(1==0)\nE               AssertionError: assert False\nE               \nE               test_xdist.py:53: AssumptionFailure\nE               >>\tpytest.assume(False)\nE               AssertionError: assert False\n\nD:\\Miniconda\\Lib\\site-packages\\six.py:718: FailedAssumption"}, "start": 1701430294728, "stop": 1701430294759, "uuid": "0a0b16f6-78a8-4fda-b95d-7aac9463fdf8", "historyId": "66253454655bb437b23c61c3063a2fd7", "testCaseId": "66253454655bb437b23c61c3063a2fd7", "fullName": "test_xdist#test_simple_assume10", "labels": [{"name": "suite", "value": "test_xdist"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "14004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_xdist"}]}